README.md

```diff
@@ -63,14 +63,16 @@ After you're finished please remove all the comments and instructions!
 
```

Comment:
- [ ] It would be nice if you customize your readme as suggested by the previous reviewer to make your readme descriptive and professional 👍 Kindly do the following things ⬇️

1. First remove all the section descriptions which are unnecessary and also remove all the other unnecessary information from your readme file 👍
2. Second insert your data to the readme sections especially Key Feature and Future Feature 👍